### ISSUE TITLE
thumbor-doctor warns on uninstalled ffmpeg

### ISSUE THREAD

**Issue by mvhirsch** [thumbor/thumbor#1387]
Hi,

I really love thumbor and currently I'm trying to build a dockerized version of v7 (see https://github.com/mvhirsch/thumbor-docker/issues/5). `thumbor-doctor` is a really helpful tool for this! Thanks!

But I noticed, it warns if `ffmpeg` could not be found. At the moment I do not want to install `ffmpeg` for image size and because [gifv optimizer as far as I understood isn't activated by default](https://github.com/thumbor/thumbor/blob/b2d26674668dc724a4f9450cd0e221153265d9bc/thumbor/config.py#L579), correct?

That said `ffmpeg` isn't needed by default, but `thumbor-doctor` will attempt to fail. Could you please add a check if `ffmpeg` needs to be installed only if the said optimizer is activated?

Thanks!

**Comment by mvhirsch** [comment]
Same's for `jpegtran` actually :sweat_smile:

**Comment by heynemann** [comment]
You can just ignore those. These are things we suggest users include as they will optimize thumbor's usage. I guess we could check the config before letting the user know they should have it installed. But I think it is a source of help for users that do not know about either ffmpeg usage or optimizers. WDYT?

**Comment by mvhirsch** [comment]
> I guess we could check the config before letting the user know they should have it installed.

You definitely should. Think of it this way: no system is asking you to install printer drivers if no printer is connected. Even if you do a system diagnose.

`thumbor-doctor` should not warn about a missing dependency, the user (on purpose) isn't going to use. All I'm asking is something like this:

```
# pseudo code
if gifv-optimizer is actived and ffmpeg is not installed
  warn user
  exit code 1
endif
``` 

Also the exit code should not return 1 in case, it's expecting normal behavior (not using optimizer and no installed dependency still leads to a fully operational thumbor installation).

I'm not a python developer, so maybe you could help me by pointing out how I may wrap up that logic?

**Comment by heynemann** [comment]
WDYT about #1393?

**Comment by mvhirsch** [comment]
+1 for `http_loaderer` :joy:

**Comment by heynemann** [comment]
Released under 7.0.3.
